content/rs/clusters/monitoring/console-metrics-definitions.md

```diff
@@ -18,68 +18,96 @@ aliases: /rs/administering/monitoring-metrics/definitions/
-|  Values in Flash | Database ROF, Shard ROF | Number of keys with values stored in Flash, not including replication |
-|  Values in RAM | Database ROF, Shard ROF | Number of keys with values stored in RAM, not including replication |
-| **What we can calculate and what we should know from the above** |  |  |
-| Redis on Flash average key size (+ overhead) |  | Redis on Flash keeps all the keys and their overheads in RAM. Use the following steps to calculate the average size of a single key (including its overhead):<br>1. `total_keys_size` = `non_values_ram_overhead` * `used_ram`<br>2. `avg_key_size` = `total_keys_size` / (`total_keys` * 2) |
-| Redis on Flash average value size, when the value is in RAM |  | Use following steps to calculate the average size of a value in RAM:<br>`total_size_of_values_in_ram` = (1 – `non_values_ram_overhead`) * `used_ram` `avg_value_size_in_ram` = `total_szie_of_values_in_ram` / (`values_in_ram` * 2) |
-| Redis on Flash average value size, when the value is in Flash |  | This is calculated as: used_flash / values_in_flash<br>**Note** – Values in Flash are serialized and are always smaller than in RAM |
-| In Redis on Flash how come:<br>`value_in_ram` + `value_in_flash` > `total_keys` |  | Calculated as: `used_flash` / `flash_limit` |
+|  % Values in RAM | Database RoF, Shard RoF | Percent of keys whose values are stored in RAM |
+|  Flash bytes/sec | Cluster RoF, Node RoF, Database RoF, Shard RoF | Read+write bytes per second on flash storage device |
+|  Flash fragmentation | Database RoF, Shard RoF | Ratio between the (logical) used flash memory and the actual physical flash that is used |
+|  Flash IOPS | Cluster RoF, Node RoF | Rate of Input/Output operations per second on flash storage device |
+|  Flash KV ops | Node RoF | Rate of operations on flash key values (read + write + del) per second |
+|  Flash ops/sec | Database RoF, Shard RoF | Rate of operations on flash key values (read + write + del) per second |
+|  Free flash | Cluster RoF, Node RoF | Free space on flash storage |
+|  RAM dataset overhead | Database RoF, Shard RoF | Percentage of the RAM limit that is used for anything other than values, such as key names, dictionaries and other overheads |
+|  RAM hit ratio | Database RoF, Shard RoF | Ratio of requests processed directly from RAM to total number of requests processed |
+|  RAM limit | Database RoF | RAM limit in bytes |
+|  RAM usage | Database RoF | Percentage of the RAM limit usage |
+|  RAM:Flash access ratio | Database RoF, Shard RoF | Ratio between logical Redis key value operations and actual flash key value operations |
+|  Used Flash | Database RoF, Shard RoF | Total RAM used to store values in Flash |
+|  Used RAM | Database RoF, Shard RoF | Total size of data stored in RAM, including keys, values, overheads and including replication, if enabled |
+|  Values in Flash | Database RoF, Shard RoF | Number of keys with values stored in Flash, not including replication |
```

Comment:
Off the top of my head, I'm not sure if the admin console capitalizes "flash" in the actual metric names, but the descriptions have some inconsistencies around the capitalization of flash storage.
```suggestion
|  Values in Flash | Database RoF, Shard RoF | Number of keys with values stored in flash, not including replication |
```